docs/openstack/backend_services_deployment.md

```diff
@@ -209,6 +262,7 @@ podified OpenStack control plane services.
               metadata:
                 annotations:
                   metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
```

Comment:
probably we should not be using this @gibizer @stuggi ?

Reply:
I think we need it. And it is fairly similar to what is documented e.g. for keystone https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs/openstack/keystone_adoption.md#procedure---keystone-adoption

Reply:
ok I'll if this works w/o that label, and update by the testing results

Reply:
@stuggi @gibi this works for my testing w/o the allow-shared-ip label. That looks correct as we are not going to share those VIPs

Reply:
it depends if you want to document to use a VIP which is shared with other services or not. I'll work in both cases, just different VIP being selected.